include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -638,7 +641,9 @@ MultigridPreconditionerBase<dim, Number>::initialize_affine_constraints(
 
   // collect all boundary functions and translate to format understood by
   // deal.II to cover all boundaries at once
-  dealii::Functions::ZeroFunction<dim, MultigridNumber> zero_function;
+  dealii::Functions::ZeroFunction<dim, MultigridNumber> zero_function(
+    dof_handler.get_fe().n_components());
+  ;
```

Comment:
There is an extra `;` here.

Reply:
ah. Do you want to fix this in a separate PR?

Reply:
will do.